Formula/spirv-tools.rb

```diff
@@ -50,8 +52,19 @@ def install
 
   test do
     cp libexec/"examples"/"main.cpp", "test.cpp"
+
+    args = []
+
+    on_macos do
+      args = "-lc++"
+    end
```

Comment:
```suggestion
    args = ["-lc++"]
```
To fix the audit, try this, along with removing the `args = []` above.

Reply:
Ah that's the trick!  Thank you!